elements/pl-matrix-component-input/pl-matrix-component-input.mustache

```diff
@@ -0,0 +1,101 @@
+
+{{#question}}
+<script>
+    $(function() {
+        $('#pl-matrix-components-input-{{uuid}} [data-toggle="popover"]').popover({
+            container: 'body',
+            template: '<div class="popover pl-matrix-components-input-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>',
+        });
+    });
+</script>
+<div id="pl-matrix-components-input-{{uuid}}" class="input-group">
+    <span class="input-group-prepend">
+        {{#label}}<label class="input-group-text" id="basic-addon2">{{label}}</label>{{/label}}
+    </span>
+    &nbsp;{{{input_array}}}&nbsp;
+    <div class="input-group-append">
+        <a class="btn btn-light border" role="button" data-toggle="popover" data-html="true" title="Matrix" data-content="{{info}}" data-placement="auto" data-trigger="focus" tabindex="0">
+            <i class="fa fa-question-circle" aria-hidden="true"></i>
+            {{#correct}}&nbsp;<span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>{{/correct}}
+            {{#partial}}&nbsp;<span class="badge badge-warning"><i class="fa fa-circle-o" aria-hidden="true"></i> {{partial}}%</span>{{/partial}}
+            {{#incorrect}}&nbsp;<span class="badge badge-danger"><i class="fa fa-times" aria-hidden="true"></i> 0%</span>{{/incorrect}}
+
+        </a>
+    </div>
+</div>
+{{/question}}
+
+{{#submission}}
+{{#parse_error}}
+<script>
+    $(function(){
+        $('#pl-matrix-component-input-submission-{{uuid}} [data-toggle="popover"]').popover({
+            container: 'body',
+            template: '<div class="popover pl-matrix-component-input-popover" role="tooltip"><div class="arrow"></div><h3 class="popover-header"></h3><div class="popover-body"></div></div>',
+        });
+    });
+</script>
+<span id="pl-matrix-component-input-submission-{{uuid}}">
+    {{#label}}<span>{{label}}</span>{{/label}}
+    <a class="btn btn-secondary btn-sm" role="button" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" title="Error" tabindex="0" data-content="{{parse_error}}{{#raw_submitted_answer}}<hr><h5><small class='text-uppercase'>What you submitted</small></h5><pre>{{raw_submitted_answer}}</pre>{{/raw_submitted_answer}}">INVALID
+      <span><i class="fa fa-question-circle" aria-hidden="true"></i></span>
+    </a>
+</span>
+{{/parse_error}}
+{{^parse_error}}
+{{#label}}<span>{{label}}</span>{{/label}}
+<samp>{{a_sub}}</samp>
+{{#correct}}<span class="badge badge-success"><i class="fa fa-check" aria-hidden="true"></i> 100%</span>{{/correct}}
+{{#partial}}<span class="badge badge-warning"><i class="fa fa-circle-o" aria-hidden="true"></i> {{partial}}%</span>{{/partial}}
```

Comment:
I think `fa-circle-o` doesn't exist? See https://fontawesome.com for the FA icons.

Reply:
I am confused what do you mean by "doesn't exist". `fa-circle-o` is used in many other elements, such as `pl-number-input`, or `pl-matrix-input`. Am I missing something?

Reply:
I see the image below, where the icon with an explanation mark indicates that the requested icon was not found. Do you see something different?

![2018-06-19 at 17 32](https://user-images.githubusercontent.com/875833/41627695-f136688a-73e6-11e8-9faf-c2e9a55e83ee.png)

Reply:
I see the same thing, and I always thought it was a weird icon, blinking different than others. But as I said, it appears in other elements, so I thought I should just keep it consistent (and maybe there was a reason for it). What do you suggest? I can just choose another icon, if you want.

Reply:
I think maybe it's supposed to be https://fontawesome.com/icons/circle?style=regular ?

I guess the other elements are also wrong in that case. Want to submit a PR to fix them? :-) @tbretl might be able to confirm this as well.

Reply:
okay, I will use the suggested icon for this element. Once @tbretl gives his input, I could also submit a PR for the other elements.

Reply:
Matt is correct, the upgrade from FA4 to FA5 is why a change is needed:
https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4